### PR TITLE
Tidy up issue template

### PR DESCRIPTION
**Pull request by aanm**
Read per commit basis

### CODE REVIEW

**Review by qmonnet — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -24,13 +24,13 @@ body:
         **NOTE: If your version is NOT listed then please upgrade before opening the GH issue. Thank you**
       options:
         # renovate: datasource=github-tags depName=cilium/cilium
-        - 'v1.16.0-rc.1 or higher'
+        - 'higher than v1.16.0-rc.1 and lower than v1.16.99'
```
Comment (joestringer) [inline .github/ISSUE_TEMPLATE/bug_report.yaml]: We don't use the versions `vX.Y.99` any more, this new version seems more confusing that than the existing version. Should this just say `lower than v1.17.0`? (Same for below)

**Review by joestringer — COMMENTED** [review]
Minor nit, otherwise lgtm

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 1 above)
Comment (aanm) [reply .github/ISSUE_TEMPLATE/bug_report.yaml]: Oh, good idea. Let's try with that.

**Review by viktor-kurchenko — APPROVED** [review]
LGTM

**Pull request merged**